services/common/src/components/modalContent/config.js

```diff
@@ -0,0 +1,45 @@
+// import AddReportModal from "@/components/modalContent/reports/AddReportModal";
+import EditReportModal from "@mds/common/components/modalContent/reports/EditReportModal";
+// import AddVarianceModal from "@/components/modalContent/variances/AddVarianceModal";
```

Comment:
I like having a shared config for this, but I think we just delete everything not being used, and add them back in as needed. (Honestly would rather get rid of the config altogether, but I don't think that's happening right away)

Reply:
Removed as these became unnecessary after addressing all the comments.